src/mlpack/tests/CMakeLists.txt

```diff
@@ -138,6 +138,7 @@ add_executable(mlpack_test
   main_tests/random_forest_test.cpp
   main_tests/softmax_regression_test.cpp
   main_tests/sparse_coding_test.cpp
+  main_tests/hmm_training_tests.cpp
```

Comment:
I know this is picky... do you think we could name the file `hmm_train_test.cpp` so it matches the names of the other files?  Thanks!